### PR TITLE
Milestone 3: Mobile-Version-2.0

### PR DESCRIPTION
**Pull request by skabeo**
In this milestone:

- Added HTML file to represent content of the web page
- Added a styles folder for layout and design of the web page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+            <li><img src="img/Rectangle 70.svg"></li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+
+        <div class="button-container">
+          <button type="button" class="button">See Project</button>
+        </div>
+      </div>
+      <div>
```
Comment (divinecharlotte) [inline index.html:70]: - [ ] I can see you used a grid on the work section which is good but still there is something you missed out kindly consider checking on this requirement again  👇 
> - In order to lay out the elements in the "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card. 
 and try to display your single card using grid

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @skabeo  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive Redame✔️ 
- Descriptive commit messages✔️ 

## Required Changes ♻️

_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by skabeo** [comment]
Hello Charlotte,

I used the CSS grid for the container layout as the instructions stated and
also used flexbox for list items. Please any advice or comment on how to
work around it?

Thank you

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @skabeo,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**